### PR TITLE
GEODE-3392: Converts CacheStatistics to std::chrono.

### PR DESCRIPTION
**Pull request by pivotal-jbarrett**
- Convert .NET CacheStatistics to DateTime.

### CODE REVIEW

File: cppcache/src/ExpiryTaskManager.cpp [hunk 1]
```diff
@@ -61,20 +61,6 @@ long ExpiryTaskManager::scheduleExpiryTask(ACE_Event_Handler* handler,
   return m_reactor->schedule_timer(handler, 0, expTimeValue, intervalValue);
 }
 
-long ExpiryTaskManager::scheduleExpiryTask(
```
Comment (dgkimura) [inline cppcache/src/ExpiryTaskManager.cpp]: I thought we were trying to get rid of the other scheduleExpiryTask above that took `uint32_t`?

File: clicache/test/TimeUtilsTests.cpp [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		//[TestInitialize()]
+		//void MyTestInitialize() {};
+		//
+		//Use TestCleanup to run code after each test has run
+		//[TestCleanup()]
+		//void MyTestCleanup() {};
+		//
+		#pragma endregion 
+
+		[TestMethod]
```
Comment (dgkimura) [inline clicache/test/TimeUtilsTests.cpp]: Nit - mixing spaces/tabs in this file

File: cppcache/integration-test/CacheHelper.cpp [hunk 3]
```diff
@@ -494,26 +494,30 @@ std::shared_ptr<Pool> CacheHelper::createPool2(
...
           pool->getName());
-  LOGINFO("getFreeConnectionTimeout: %d", pool->getFreeConnectionTimeout());
+  LOGINFO("getFreeConnectionTimeout: %s",
+          to_string(pool->getFreeConnectionTimeout()).c_str());
   LOGINFO("getLoadConditioningInterval: %d",
-          pool->getLoadConditioningInterval());
+          to_string(pool->getLoadConditioningInterval()).c_str());
   LOGINFO("getSocketBufferSize: %d", pool->getSocketBufferSize());
-  LOGINFO("getReadTimeout: %d", pool->getReadTimeout());
+  LOGINFO("getReadTimeout: %d", to_string(pool->getReadTimeout()).c_str());
```
Comment (dgkimura) [inline cppcache/integration-test/CacheHelper.cpp]: I think format should be %s on all the ones you changed to pass char*.

File: clicache/test/TimeUtilsTests.cpp (hunk 2 above)
Comment (pivotal-jbarrett) [reply clicache/test/TimeUtilsTests.cpp]: Grrr! Let me look into what VS is doing there. Wish we could find a good C++/CLR formatter.

File: cppcache/integration-test/CacheHelper.cpp (hunk 3 above)
Comment (pivotal-jbarrett) [reply cppcache/integration-test/CacheHelper.cpp]: Yes great catch!

File: cppcache/src/ExpiryTaskManager.cpp (hunk 1 above)
Comment (pivotal-jbarrett) [reply cppcache/src/ExpiryTaskManager.cpp]: Still some internal usage. Focusing on external APIs first.

**Comment by pivotal-jbarrett** [comment]
@dgkimura Please have another look.

**Pull request merged**